app/javascript/components/Greetings.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetings } from '../redux/greetings/greetingsSlice';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const { messages, isLoading } = useSelector((store) => store.greeting);
+
```

Comment:
- [ ] After solving the other error, the following error appeared. Kindly make sure to solve all errors and that the app is working as intended.

![image](https://github.com/meryemsanem/hello-rails-react/assets/62290545/4b17d024-a725-4678-8eaa-32966705951f)